lib/block-supports/layout.php

```diff
@@ -573,128 +573,132 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
+				if ( ! in_array( $parent_column_unit, array( 'px', 'rem', 'em' ), true ) ) {
+					$parent_column_unit = 'rem';
+				}
 			}
+
+			/*
+			 * A default gap value is used for this computation because custom gap values may not be
+			 * viable to use in the computation of the container query value.
+			 */
+			$default_gap_value     = 'px' === $parent_column_unit ? 24 : 1.5;
+			$container_query_value = $highest_number * $parent_column_value + ( $highest_number - 1 ) * $default_gap_value;
+			$container_query_value = $container_query_value . $parent_column_unit;
+			// If a span is set we want to preserve it as long as possible, otherwise we just reset the value.
+			$grid_column_value = $column_span ? '1/-1' : 'auto';
+
+			$child_layout_styles[] = array(
+				'rules_group'  => "@container (max-width: $container_query_value )",
+				'selector'     => ".$container_content_class",
+				'declarations' => array(
+					'grid-column' => $grid_column_value,
+				),
+			);
 		}
 
 		/*
-		 * A default gap value is used for this computation because custom gap values may not be
-		 * viable to use in the computation of the container query value.
-		 */
-		$default_gap_value     = 'px' === $parent_column_unit ? 24 : 1.5;
-		$container_query_value = $highest_number * $parent_column_value + ( $highest_number - 1 ) * $default_gap_value;
-		$container_query_value = $container_query_value . $parent_column_unit;
-		// If a span is set we want to preserve it as long as possible, otherwise we just reset the value.
-		$grid_column_value = $column_span ? '1/-1' : 'auto';
-
-		$child_layout_styles[] = array(
-			'rules_group'  => "@container (max-width: $container_query_value )",
-			'selector'     => ".$container_content_class",
-			'declarations' => array(
-				'grid-column' => $grid_column_value,
-			),
+		* Add to the style engine store to enqueue and render layout styles.
+		* Return styles here just to check if any exist.
+		*/
+		$child_css = gutenberg_style_engine_get_stylesheet_from_css_rules(
+			$child_layout_styles,
+			array(
+				'context'  => 'block-supports',
+				'prettify' => false,
+			)
 		);
-	}
-
-	/*
-	 * Add to the style engine store to enqueue and render layout styles.
-	 * Return styles here just to check if any exist.
-	 */
-	$child_css = gutenberg_style_engine_get_stylesheet_from_css_rules(
-		$child_layout_styles,
-		array(
-			'context'  => 'block-supports',
-			'prettify' => false,
-		)
-	);
 
-	if ( $child_css ) {
-		$outer_class_names[] = $container_content_class;
+		if ( $child_css ) {
+			$outer_class_names[] = $container_content_class;
+		}
 	}
 
-	// Prep the processor for modifying the block output.
-	$processor = new WP_HTML_Tag_Processor( $block_content );
+		// Prep the processor for modifying the block output.
+		$processor = new WP_HTML_Tag_Processor( $block_content );
 
-	// Having no tags implies there are no tags onto which to add class names.
+		// Having no tags implies there are no tags onto which to add class names.
```

Comment:
The whitespace here and below before `if ( ! $block_supports_layout && ! empty( $outer_class_names ) ) {` appears to be off (extra indentation).

Reply:
Thanks for noticing! Not sure what happened here, the auto-formatter usually knows what it's doing 🤔